styles/style-main.css

```diff
@@ -0,0 +1,95 @@
+.navbar {
+  display: flex;
+  justify-content: center;
+  background-color: beige;
+}
+
+.navbar ul {
+  display: flex;
+  align-self: center;
+  width: 40vw;
+  padding: 10px;
+  justify-content: center;
+}
+
+.navbar ul > * {
+  flex: 1 1 25%;
+  list-style: none;
+  text-align: center;
+  font-size: 30px;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+a {
+  text-decoration: none;
+}
+
+main {
+  display: block;
+}
+
+section {
+  display: flex;
+}
+
+.headline-left {
+  display: flex;
+  flex-direction: column;
+  flex: 1 1 60%;
+}
+
+.headline-name {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-weight: 600;
+  font-size: 7em;
+  margin-left: 150px;
+  margin-top: 50px;
+  margin-bottom: 50px;
+}
+
+.headline-bio {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 30px;
+  margin-left: 150px;
+  margin-bottom: 50px;
+}
+
+#start-collaboration {
+  background-color: rgb(14, 14, 68);
+  color: white;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
```

Comment:
- [ ] The font family here is also Poppins, but with a weight of 500, so it'd be better if you also changed it
![image](https://user-images.githubusercontent.com/73140461/143500718-29f03c6e-77fc-4098-aee8-d9d89015ee61.png)